.github/workflows/test.yml

```diff
@@ -1,107 +1,110 @@
 name: Continuous integration
```

Comment:
While we are at it could we abbreviate this so it doesn't take all of the width on mobile?

```suggestion
name: CI
```